ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/AuthEndpoint.java

```diff
@@ -190,15 +196,16 @@ private NewCookie createSessionCookie(String sessionId) {
   @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
   public Response postSelectIdp(
       @CookieParam("session_id") String sessionId,
-      @FormParam("identityProvider") String identityProvider) {
+      @FormParam("identityProvider") String identityProvider,
+      @HeaderParam("Accept-Language") @DefaultValue("en-GB") String acceptLanguage) {
 
     if (identityProvider == null || identityProvider.isBlank()) {
-      return badRequest("No identity provider selected. Please go back.");
+      return badRequest("No identity provider selected. Please go back.", acceptLanguage);
```

Comment:
I don't understand your logic here. You are inserting a fixed english error message into a localized bundle?

Reply:
Yeah right this is wrong. My question then is, should we map all the message coming from the several bad request to be also in German?